packages/deployment/scripts/integration-test.sh

```diff
@@ -15,6 +15,10 @@ popd
 mkdir -p "$NETWORK_NAME/setup"
 cd "$NETWORK_NAME/setup"
 
+# Use the OpenTelemetry tracing SLOGSENDER to test it against real slog data,
+# plus keep a recorder.
+export SLOGSENDER=@agoric/telemetry/src/otel-and-flight-recorder.js
```

Comment:
Wondering if we shouldn't only pass it to the setup bootstrap script so that it applies only the the validators and not the loadgen. It should hurt to do both, but at least we exercise variety

Reply:
Good point.  Changed.